perllib/FixMyStreet/Script/Inactive.pm

```diff
@@ -0,0 +1,79 @@
+package FixMyStreet::Script::Inactive;
+
+use v5.14;
+use warnings;
+
+use Moo;
+use CronFns;
+use FixMyStreet;
+use FixMyStreet::Cobrand;
+use FixMyStreet::DB;
+use FixMyStreet::Email;
+
+has anonymize => ( is => 'ro' );
+has email => ( is => 'ro' );
+has verbose => ( is => 'ro' );
+
+has cobrand => (
+    is => 'lazy',
+    default => sub {
+        my $base_url = FixMyStreet->config('BASE_URL');
+        my $site = CronFns::site($base_url);
+        FixMyStreet::Cobrand->get_class_for_moniker($site)->new;
+    },
+);
+
+sub users {
+    my $self = shift;
+    $self->anonymize_users;
+    $self->email_inactive_users;
+}
```

Comment:
I wonder about this having some sort of summary mode in between "say nothing" and "print out everything I am doing" might be an idea. Mostly as a reassurance that something happened but without screeds of output. Probably not as I guess this will mostly be run by cron and no one wants emails about things that just work.